vectors_deep1b/challenges/default.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "index-and-query",
+  "description": "indexes vectors, force merges to a single segment, queries",
+  "default": true,
+  "schedule": [
+    {
+      "operation": {
+        "operation-type": "delete-index"
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "create-index"
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "cluster-health",
+        "request-params": {
+          "wait_for_status": "green"
+        },
+        "retry-until-success": true
+      }
+    },
+    {
+      "operation": "index-append",
+      "warmup-time-period": 0,
+      "clients": {{bulk_indexing_clients | default(8)}}
+    },
+    {
```

Comment:
See the comment below regarding the `_knn_search` operation; I assume the operations below (force-merge etc.) are intended for use when the `_knn_search` operation gets added, but for now, for indexing only they are not needed right?

Reply:
We have noticed that while indexing knn vectors, `force-merge` takes substantial time, and we are working on optimizing it, so we would like to start tracking it right now, even before running `_knn_search`.